integration-cli/docker_utils.go

```diff
@@ -463,6 +465,32 @@ func (d *Daemon) waitRun(contID string) error {
 	return waitInspectWithArgs(contID, "{{.State.Running}}", "true", 10*time.Second, args...)
 }
 
+func (d *Daemon) getBaseDeviceSize(c *check.C) int64 {
+
```

Comment:
can you maybe clean this up a bit, random carrige returns and vars should not be starting with a capital